### PR TITLE
Ruby Capstone Project: Catalog of My Things

### PR DESCRIPTION
**Pull request by shahira-sadat**
- There is a `main.rb` file as console app entry-point which at the start:
  - Present the user with a list of options to perform.
  - Let users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

- The `Catalog of My Things`  app has the following options:
  - List all books
  - List all music albums
  - List of games
  - List all genres
  - List all labels
  - List all authors
  - Add a book
  - Add a music album
  - Add a game

- All data preserves by saving collections in `.json` files.
- Created a `schema.sql` file with tables that will be analogical to the structure of the classes in app. 

[Video](https://drive.google.com/drive/folders/1EWHotmbfvoPbZcAKUuqrzTyh_Ylp2fPg?usp=sharing)

### CODE REVIEW

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Alzubair980, @shahira-sadat and @wayungi  
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️

- [x] Kindly record another video using zoom or another platform as some requirements for the [project documentation](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md#project-documentation) is not yet met because the audio for the other two team members is off, please after recording another video following all the requirements, kindly make sure that you watch to check that there is clear audio for all team members👍🏿 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,47 @@
...
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
+## video
+
+[LINK](https://drive.google.com/drive/u/1/folders/1EWHotmbfvoPbZcAKUuqrzTyh_Ylp2fPg)
```
Comment (danmainah) [inline README.md:47]: - [x] @team great work on your presentation, however, you forgot the project description part, where you explain the purpose of the project. Kindly add a video describing your project's purpose.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @shahira-sadat , @wayungi and @Alzubair98 , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Alzubair98) [reply README.md:47]: fixed

**Review by DammyShittu — APPROVED** [review]
Hi team,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**